akka-http/server-playjson/src/main/scala/endpoints/akkahttp/server/playjson/JsonSchemaEntities.scala

```diff
@@ -0,0 +1,38 @@
+package endpoints.akkahttp.server.playjson
+
+import akka.http.scaladsl.marshalling.{Marshaller, ToResponseMarshaller}
+import akka.http.scaladsl.server.Directives
+import akka.http.scaladsl.unmarshalling.Unmarshaller
+import endpoints.akkahttp.server
+import endpoints.algebra.Documentation
+import endpoints.algebra
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport
+import endpoints._
+
+import akka.http.scaladsl.marshalling.{Marshaller, ToResponseMarshaller}
+import akka.http.scaladsl.unmarshalling.Unmarshaller
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport
+import endpoints.algebra.Documentation
+
+/**
+  * Interpreter for [[algebra.JsonEntities]] that uses Play JSON [[play.api.libs.json.Reads]] to decode
+  * JSON entities in HTTP requests, and [[play.api.libs.json.Writes]] to build JSON entities
+  * in HTTP responses.
+  *
+  * @group interpreters
+  */
+trait JsonSchemaEntities extends server.Endpoints with algebra.JsonSchemaEntities with endpoints.playjson.JsonSchemas {
```

Comment:
Is it possible to also add tests for that trait? IIRC, the same tests could be reused with both `akka-http-server-circe` and `akka-http-server-playjson`, you “just” need to wire the right traits together! (but I might be wrong because I see that there are no tests for the `play-server` interpreters…)

Reply:
I will have a look into it

Reply:
Can you have a look?

Reply:
I don't see why Scala 2.11 get's an unimeplemented contramap function...

Reply:
I can have a look this evening.

Reply:
I think this is because `akka-http-play-json` 1.25.2 depends on play-json 2.7, but `endpoints-algebra-playjson` depends on play-json 2.6. Changing the version of `akka-http-play-json` to 1.24.3 fixed the issue.